OpenEdXMobile/src/main/java/org/edx/mobile/social/facebook/UserRequest.java

```diff
@@ -0,0 +1,24 @@
+package org.edx.mobile.social.facebook;
+
+import android.os.Bundle;
+
+import com.facebook.AccessToken;
+import com.facebook.GraphRequest;
+import com.facebook.HttpMethod;
+
+public class UserRequest {
+    private static final String ME_ENDPOINT = "/me";
+
+    public static void makeUserRequest(GraphRequest.Callback callback) {
+        final Bundle params = new Bundle();
+        params.putString("fields", "picture,name,id,email,permissions");
```

Comment:
Previously we had: `params.putString("fields", "email,id,name");`
Why have we added `picture` and `permissions` param?

Reply:
removed!